protocol/app/msgs/unsupported_msgs.go

```diff
@@ -18,6 +20,41 @@ var (
 		"/cosmos.gov.v1beta1.MsgSubmitProposal":         &govbeta.MsgSubmitProposal{},
 		"/cosmos.gov.v1beta1.MsgSubmitProposalResponse": nil,
 
+		// cosmwasm
+		"/cosmwasm.wasm.v1.AcceptedMessageKeysFilter":                  nil,
+		"/cosmwasm.wasm.v1.AcceptedMessagesFilter":                     nil,
+		"/cosmwasm.wasm.v1.AllowAllMessagesFilter":                     nil,
+		"/cosmwasm.wasm.v1.ClearAdminProposal":                         nil,
+		"/cosmwasm.wasm.v1.CombinedLimit":                              nil,
+		"/cosmwasm.wasm.v1.MaxCallsLimit":                              nil,
+		"/cosmwasm.wasm.v1.MaxFundsLimit":                              nil,
+		"/cosmwasm.wasm.v1.MsgClearAdmin":                              &wasm.MsgClearAdmin{},
```

Comment:
Believe this message is [needed](https://github.com/CosmWasm/wasmd/blob/c951068274fd921f3501b0dff144eba0e6aac13b/proto/cosmwasm/wasm/v1/proposal_legacy.proto#L226) for governance module to clear admin on a contract. Also since this message is removing privileges rather than adding it, I think it's safe to support

Reply:
For all changes to this list, pls update https://www.notion.so/dydx/CosmWasm-Implementation-Spec-aa96d7ae25c64ccaadc1f61827c69e73?pvs=4#8d42683414de4683a77ac97ecd1e9dc5 accordingly